src/internal.c

```diff
@@ -12858,6 +12858,7 @@ int ProcessReply(WOLFSSL* ssl)
 
             /* see if sending SSLv2 client hello */
             if ( ssl->options.side == WOLFSSL_SERVER_END &&
+                 !IsAtLeastTLSv1_3(ssl->version) &&
```

Comment:
How can we guarantee a client won't send an SSLv2 client hello?  Shouldn't we handle the old hello case?

Reply:
Better fix put up.
Please review this.